airbyte-cdk/python/airbyte_cdk/sources/declarative/auth/declarative_authenticator.py

```diff
@@ -3,20 +3,32 @@
 #
 
 from dataclasses import InitVar, dataclass
-from typing import Any, Mapping
+from typing import Any, Mapping, MutableMapping, Union
 
 from airbyte_cdk.sources.streams.http.requests_native_auth.abstract_token import AbstractHeaderAuthenticator
 
 
 @dataclass
-class DeclarativeAuthenticator:
+class DeclarativeAuthenticator(AbstractHeaderAuthenticator):
     """
     Interface used to associate which authenticators can be used as part of the declarative framework
     """
 
+    def get_request_params(self) -> Union[MutableMapping[str, Any], None]:
+        """HTTP request parameter to add to the requests"""
+        return None
```

Comment:
is there value in returning `None` instead of `{}`?

Reply:
Not really, I'll simplify